teams/Dependancies.toml

```diff
@@ -0,0 +1,49 @@
+[[dependency]]
+org = "ballerina"
+name = "log"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "regex"
+version = "0.7.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "crypto"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "time"
+version = "2.0.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "xmldata"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "os"
+version = "0.8.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "io"
+version = "0.6.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "url"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "mime"
+version = "1.1.0-beta.1"
```

Comment:
Since the latest version is beta.2, Change versions to beta.2

Reply:
Changed thanks